rse-project.Rmd

```diff
@@ -692,6 +692,53 @@ Breaking this down:
 > Documentation that serves these two groups well
 > will almost certainly serve the needs of strangers as well.
 
+These additional rules are taken from [@Good2014,@Mich2015,@Hart2016,@Zook2017]:
+
+-   **Keep raw data raw.**
```

Comment:
Raw data may also just the data as you received it or downloaded it from a repository.  Retaining an unchanged version is important for reproducibility, and in case you use the wrong read/write mode and blow away your working copy.